daemon/container_linux.go

```diff
@@ -0,0 +1,22 @@
+//+build !windows
+
+package daemon
+
+import (
+	"github.com/docker/docker/container"
+)
+
+func (daemon *Daemon) fixSecurityOptions(container *container.Container) error {
+	container.AppArmorProfile = "" //we don't care about the previous value.
+	if err := parseSecurityOpt(container, container.HostConfig); err != nil {
+		return err
+	}
+	if daemon.apparmorEnabled && !container.HostConfig.Privileged {
```

Comment:
I think empty would do. `unconfined` implies that special privileged profile was applied(at least to me).